packages/sites/ortho-site/webapp/wdkCustomization/js/client/records/Sequences.tsx

```diff
@@ -331,12 +342,17 @@ export function RecordTable_Sequences(
     [mesaColumns]
   );
 
-  const handleSpeciesSelection = useCallback(
-    (species: string[]) => {
-      setSelectedSpecies(species);
+  const handleSpeciesSelection = useCallback((species: string[]) => {
```

Comment:
memoised a couple of handlers, just in case (they are hook dependencies downstream)